data/INRA/data5/end_of_alcoholic_fermentation.ttl

```diff
@@ -0,0 +1,7294 @@
+   bdg:date    "2016-09-19"^^xsd:Date;
+   bdg:GlucoseFructoseConcentration    "0.08"^^xsd:decimal .
+
+<data/winemaking/PechRouge/EndFermentationMonitoring/2016/c602c721-4aa4-4f86-afb6-db4d2bbd2abb> a qb:Observation;
+   qb:dataSet  <data/winemaking/PechRouge/EndFermentationMonitoring/2016>;
+   bdg:fermentationTank    <fermentationTank/f3100873-e2b5-4db0-b34c-a99de14fbe38>;
+   bdg:wineExperimentalModality    <experiment/2016/2016_INERTIS_VIO_05>;
+   bdg:date    "2016-09-19"^^xsd:Date;
+   bdg:GlucoseFructoseConcentration    "0.05"^^xsd:decimal .
+
+<data/winemaking/PechRouge/EndFermentationMonitoring/2016/4e4a6441-3346-4d4b-a374-0e7fd6930b96> a qb:Observation;
+   qb:dataSet  <data/winemaking/PechRouge/EndFermentationMonitoring/2016>;
+   bdg:fermentationTank    <fermentationTank/a9e1dc0e-693a-45cc-b7a4-26b2f9af33fe>;
+   bdg:wineExperimentalModality    <experiment/2016/2016_PULVGLUT_GN_N>;
+   bdg:date    "2016-09-19"^^xsd:Date;
+   bdg:GlucoseFructoseConcentration    "0.06"^^xsd:decimal .
+
+<data/winemaking/PechRouge/EndFermentationMonitoring/2016/eb5f7771-2831-4c36-9a23-61f9467d6c74> a qb:Observation;
+   qb:dataSet  <data/winemaking/PechRouge/EndFermentationMonitoring/2016>;
+   bdg:fermentationTank    <fermentationTank/f67c6eda-060b-4d09-804c-2831c2fdd6be>;
+   bdg:wineExperimentalModality    <experiment/2016/2016_Vitidur_3160_RS>;
+   bdg:date    "2016-09-19"^^xsd:Date;
+   bdg:GlucoseFructoseConcentration    "0.48"^^xsd:decimal .
+
+<data/winemaking/PechRouge/EndFermentationMonitoring/2016/910e1df6-756c-4075-917b-2f78c9fd321d> a qb:Observation;
+   qb:dataSet  <data/winemaking/PechRouge/EndFermentationMonitoring/2016>;
+   bdg:fermentationTank    <fermentationTank/babbf0b3-30bc-49cc-a515-6c84c8cd28af>;
+   bdg:wineExperimentalModality    <experiment/2016/2016_ALF_SYR_01>;
+   bdg:date    "2016-09-19"^^xsd:Date;
+   bdg:GlucoseFructoseConcentration    "0.05"^^xsd:decimal .
+
+<data/winemaking/PechRouge/EndFermentationMonitoring/2016/8f82e833-5e6c-44a5-8f3f-35a34e322788> a qb:Observation;
+   qb:dataSet  <data/winemaking/PechRouge/EndFermentationMonitoring/2016>;
+   bdg:fermentationTank    <fermentationTank/c3f36d78-4a4e-444e-874e-dbe60cecd598>;
+   bdg:wineExperimentalModality    <experiment/2016/2016_ALF_SYR_02>;
+   bdg:date    "2016-09-19"^^xsd:Date;
+   bdg:GlucoseFructoseConcentration    "0.02"^^xsd:decimal .
+
+<data/winemaking/PechRouge/EndFermentationMonitoring/2016/671b6296-adac-434b-9153-b8687a5fb42c> a qb:Observation;
+   qb:dataSet  <data/winemaking/PechRouge/EndFermentationMonitoring/2016>;
+   bdg:fermentationTank    <fermentationTank/7d2b34e8-712f-4cfb-9157-916f6c3ad5db>;
+   bdg:wineExperimentalModality    <experiment/2016/2016_ALF_SYR_03>;
+   bdg:date    "2016-09-19"^^xsd:Date;
+   bdg:GlucoseFructoseConcentration    "9"^^xsd:decimal .
+
+<data/winemaking/PechRouge/EndFermentationMonitoring/2016/44a57b69-dc3a-481c-8ec0-a1081877f74a> a qb:Observation;
+   
```

Comment:
The DSD says there are 3 measures, so each Observation must have 3 measures. Instead, you have 3 sublists of Observations, each with just 1 measure. These sublists don't conform to the DSD.

Reply:
6bae82e

Reply:
I correct DSD in the 752f327 commit of #6 pull request. 
Do I need to split my dataset if there is somes times missing qb:measure value.
For example, If my first observation contains glucose and sulfuricAcid measurements and the second contains glucose, sulfuricAcid and malicAcid measurements how can I manage it in DSD for the same dataset.

Reply:
@niio972 unfortunately you need to split: https://www.w3.org/TR/vocab-data-cube/#h3_dsd-dsd "qb:componentRequired declaration may only be applied to component specifications of attributes - measures and dimensions are always required."
On the bright side, then you don't need over-specific vars like `concentrationOfMalicAcid` because you can attach the `featureOfInterest` to the dataset. cc @nikolatulechki

Reply:
The commit https://github.com/niio972/ontology/commit/07ef3fe6d3585547705eebb2afc1e80474b4f30c correct the dsd i put has you said the featureOfInterest in the dataset.